packages/neovim/src/api/Window.test.ts

```diff
@@ -105,15 +107,15 @@ describe('Window API', () => {
 
       windows = await nvim.windows;
       // Set to new split
-      [, nvim.window] = windows;
+      nvim.window = windows[1] as unknown as AsyncWindow;
```

Comment:
can this be avoided by changing the `let windows` delcaration on  line 99 to:

      let windows: Awaited<typeof nvim.windows>;

Reply:
That's look way better!